release.config.js

```diff
@@ -0,0 +1,61 @@
+module.exports = {
+  'branches': ['+([0-9])?(.{+([0-9]),x}).x', 'main', 'next', 'next-major', {name: 'beta', prerelease: true}, {name: 'alpha', prerelease: true}],
+  'plugins': [
+    '@semantic-release/commit-analyzer',
+    '@semantic-release/release-notes-generator',
+    [
+      '@semantic-release/changelog',
+      {
+        'changelogFile': 'CHANGELOG.md',
+      }
+    ],
+    [
+      '@amanda-mitchell/semantic-release-npm-multiple',
+      {
+        'registries': {
```

Comment:
This is great @jim80net ! And also gives us a short path to automating npm publishing when we need it 👍